workspaces/src/network/variants.rs

```diff
@@ -16,15 +16,15 @@ pub trait NetworkInfo {
 }
 
 #[async_trait]
-pub trait TopLevelAccountCreator {
-    async fn create_tla(
+pub trait DevAccountCreator {
```

Comment:
After a second thought, I think it is better to have two traits:

1. TopLevelAccountCreator (it is still possible to do that in the sandbox as we control the registrar account, so there is still a value to have this trait as it was before the PR)
2. SponsoredAccountCreator

Sandbox can implement both, testnet will implement only SponsoredAccountCreator, and custom wrappers can even implement it for mainnet.

Reply:
Spent time thinking and did not understand how to implement functions(https://github.com/near/near-workspaces-rs/pull/369/files#diff-2d190d038b29b514c04b33ac855123d6949ed0f3e8b5b8b7034b0de327992e79R62) , that must to get both tla and sponsored accounts. Also as the sandbox can implement both traits, how can we choose method that will be used